.gitlab/docker_common/publish_job_templates.yml

```diff
@@ -11,6 +11,8 @@
     <<: *docker_variables
     IMG_VARIABLES: ""
     IMG_SIGNING: ""
+  before_script:
+    - export COMMIT_TIME=$(git show -s --format=%ct $CI_COMMIT_SHA)
```

Comment:
does the `COMMIT_SHA` is not enough?

Reply:
The motivation for including the commit time is to make it easier for viewers of Single Machine Performance's outputs -- which will eventually include Datadog notebooks and dashboards -- to see how performance has changed over time, ideally at a glance. This option is one of the lower development cost efforts for getting at that time information for visualization purposes.